test/blocks/ost/mocks/ost-utils.js

```diff
@@ -9,8 +9,30 @@ const getConfig = () => ({
     '': { prefix: '', ietf: 'en-US', tk: 'hah7vzn.css' },
     ch_de: { prefix: '/ch_de', ietf: 'de-CH', tk: 'vin7zsi.css' },
   },
+  susiOptions: {
+    dctx_id: {
+      stage: 'dctx_id',
+      prod: 'dctx_id',
+    },
+  },
 });
 
+const getSusiOptions = () => {
```

Comment:
Is this helper still needed in the tests?

Reply:
Thanks for catching it!

Reply:
Cleaned up and added a fallback {} in setConfig